db/change_cache.go

```diff
@@ -639,16 +659,32 @@ func (c *changeCache) processEntry(change *LogEntry) base.Set {
 	}
 
 	sequence := change.Sequence
-	base.SetIfMax(c.context.DbStats.StatsDatabase(), base.StatKeyHighSeqFeed, int64(change.Sequence))
+	if change.Sequence > c.internalStats.highSeqFeed {
+		c.internalStats.highSeqFeed = change.Sequence
+	}
 
+	oldestPendingSequence := c.skippedSeqs.getOldest()
+	if oldestPendingSequence == 0 {
+		oldestPendingSequence = c.nextSequence
+	}
```

Comment:
This doesn't seem to be doing anything. `oldestPendingSequence` is never used after this.